### ISSUE TITLE
Python required version is actually 3.9+

### ISSUE THREAD

**Issue by gtesoro** [stegm/pykoplenti#22]
Hi,

First of all thanks for the API!
The README states that 3.7 is required but it seems that your dataclasses are now using 3.9+ typing style.

Cheers

**Comment by stegm** [comment]
Yes, you are correct. The README is outdated, setup.cfg is newer. Only Python 3.9+ is supported.
